src/feedback/index.css

```diff
@@ -0,0 +1,5 @@
+@import url(./Spinner.css);
+@import url(./Button.css);
+@import url(./FeedbackDialog.css);
+@import url(./FeedbackForm.css);
+@import url(./StarRating.css);
```

Comment:
Flattening out the dependency graph into a manifest like this is eventually going to hurt us. I think we should import component CSS from the component the CSS affects to avoid mistakes with forgetting to import CSS here, or removing it again when a component is no longer imported

Reply:
I'm all for that, just ran into an issue where the styles get duplicated if you for example use the button component multiple times. Happy to do it either way but we need to dedupe the styles somehow.

Reply:
Ah, a conflict with the need for inlining the styles I presume. Maybe we can save this for later then